src/routes/intro/__tests__/IntroScreen.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { fireEvent } from '@testing-library/react-native'
+import { mocked } from 'jest-mock'
+import React from 'react'
+
+import labels from '../../../constants/labels.json'
+import { useLoadDisciplines } from '../../../hooks/useLoadDisciplines'
+import useReadSelectedProfessions from '../../../hooks/useReadSelectedProfessions'
+import createNavigationMock from '../../../testing/createNavigationPropMock'
+import { getReturnOf } from '../../../testing/helper'
+import { mockDisciplines } from '../../../testing/mockDiscipline'
+import render from '../../../testing/render'
+import IntroScreen from '../IntroScreen'
+
+jest.mock('@react-navigation/native')
+jest.mock('../../../hooks/useLoadDisciplines')
+jest.mock('../../../hooks/useReadSelectedProfessions')
+
+describe('IntroScreen', () => {
+  const navigation = createNavigationMock<'Intro'>()
+
+  it('should navigate to discipline', async () => {
+    mocked(useLoadDisciplines).mockReturnValueOnce(getReturnOf(mockDisciplines))
+    mocked(useReadSelectedProfessions).mockReturnValueOnce(getReturnOf(null))
+
+    const { findByText } = render(<IntroScreen navigation={navigation} />)
+    const firstDiscipline = await findByText('First Discipline')
```

Comment:
No need for `await findByText`, `getByText` is enough

Reply:
Why? Because I mocked every async stuff and it is no longer asnyc then?

Reply:
yes, exactly.